src/utils/atomic.ts

```diff
@@ -1,3 +1,7 @@
+// @ts-ignore Stencil does not support subpath exports.
+import {defineCustomElements} from "@coveo/atomic/loader";
```

Comment:
Importantly, even with `skipLibCheck: true`, if the application imports one of our [subpath exports](https://nodejs.org/api/packages.html#subpath-exports), Stencil's embedded typescript config will throw a fit, meaning that the import should be annotated with a `@ts-ignore`.